cmake/Modules/LibAddMacros.cmake

```diff
@@ -529,7 +529,7 @@ if (BUILD_DOCUMENTATION AND RONN_LOC) # disable function when RONN_LOC is not se
 	add_custom_command(
 		OUTPUT ${OUTFILE}
 		DEPENDS ${MDFILE}
-		COMMAND ${RONN_LOC}
+		COMMAND export RUBYOPT="-Eutf-8" && ${RONN_LOC}
```

Comment:
Is maybe `RUBYOPT="-Eutf-8" ${RONN_LOC}` more portable? Or even better would be`${CMAKE_COMMAND} -E env RUBYOPT="-Eutf-8" ${RONN_LOC}` but it is not available in cmake 3.